users/serializers.py

```diff
@@ -0,0 +1,16 @@
+from django.conf                import settings
+from django.contrib.auth        import get_user_model
+
+from rest_framework.serializers import ModelSerializer
```

Comment:
django랑 rest_framework는 둘다 서드파티니 붙이시는 것 이 좋을 것 같습니다.
- python 내장모듈
- 서드파티
- 직접 구현

```suggestion
from django.conf                import settings
from django.contrib.auth        import get_user_model
from rest_framework.serializers import ModelSerializer
```

Reply:
수정되었습니다!